### PR TITLE
Update integration test readme

### PR DESCRIPTION
**Pull request by hasLeland**
The README in `cmd/_integration-tests/` had drifted out of date with the actual structure of that folder. This is an update to that README.

### CODE REVIEW

File: cmd/_integration-tests/README.md [hunk 1]
```diff
@@ -1,40 +1,39 @@
...
 
 These tests run truss against definition files and tests if the generated code
-behaves. 
+behaves. Each test imports and uses service code generated by `truss`, but that code doesn't exist by default and must be generated before each test runs.
 
-# ./http
+# ./transport
 
-The `http` directory contains tests for the generated HTTP client libraries and
+The `transport` directory contains tests for the generated HTTP client libraries and
```
Comment (hasAdamr) [inline cmd/_integration-tests/README.md]: Could you update this line to say that it "contains tests for the generated http and grpc transport libraries as well as standard Go 'net/http' requests.

File: cmd/_integration-tests/README.md [hunk 2]
```diff
@@ -1,40 +1,39 @@
...
 - Run `go test -v`
-- Runs truss again against `http/httptest.proto` (for regeneration tests)
+- Runs truss again against `transport/transport-test.proto` (for regeneration tests)
 - Run `go test -v`
 
-`http/handlers` has implemented handlers for the server. They add things
+`transport/handlers` has implemented handlers for the server. They add things
 together for the purposes of testing.
 
 `http_test.go` imports the generated code, starts up a `httptest` server with
```
Comment (hasAdamr) [inline cmd/_integration-tests/README.md]: Either remove this part or update it to say there is the setup_test for setting up a truss grpc and http server and there is http_test.go for http tests and grpc_test.go for grpc tests.

**Review by hasAdamr — APPROVED** [review]
Hooray to updating the docs! Small comments, nothing blocking though.

**Comment by hasLeland** [comment]
@hasAdamr I've made the changes you requested, so I'm merging.

**Pull request merged**